sdk/storage/Azure.Storage.DataMovement.Blobs/stress/src/Infrastructure/EnvironmentReader.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Core;
+
+namespace Azure.Storage.DataMovement.Blobs.Stress;
+
+/// <summary>
+///   Allows for reading and parsing a set of environment information in the
+///   dotenv format.
+/// </summary>
+///
+/// <seealso href="https://www.dotenv.org/"/>
+///
+internal static class EnvironmentReader
+{
+    /// <summary>
+    ///   Loads a dotenv file and parses the contents.
+    /// </summary>
+    ///
+    /// <param name="filePath">The full path, including filename, to the dotenv file.</param>
+    ///
+    /// <returns>The set of environment content parsed from the file.</returns>
+    ///
+    /// <exception cref="ArgumentException">Occurs when <paramref name="filePath"/> is <c>null</c> or empty.</exception>
+    /// <exception cref="FileNotFoundException">Occurs when <paramref name="filePath" /> does not exist.</exception>
+    /// <exception cref="FormatException">Occurs when a line of the file is malformed or the file contains duplicate environment variables.</exception>
+    ///
+    public static Dictionary<string, string> LoadFromFile(string filePath)
```

Comment:
Azure SDK doesn't have any common code for this stuff? Are we the first .NET package to stress test?

Reply:
They don't have common code for this. I pulled this from what ServiceBus did.

Reply:
Unfortunate.